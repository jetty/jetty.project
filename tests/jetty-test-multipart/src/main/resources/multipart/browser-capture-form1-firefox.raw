-----------------------------41184676334
Content-Disposition: form-data; name="user"

fireuser
-----------------------------41184676334
Content-Disposition: form-data; name="comment"

with detailed message
-----------------------------41184676334--
