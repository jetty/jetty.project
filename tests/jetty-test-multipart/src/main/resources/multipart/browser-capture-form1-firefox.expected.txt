Request-Header|Accept|text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Request-Header|Accept-Encoding|gzip, deflate
Request-Header|Accept-Language|en-US,en;q=0.5
Request-Header|Connection|keep-alive
Request-Header|Content-Length|258
Request-Header|Content-Type|multipart/form-data; boundary=---------------------------41184676334
Request-Header|Host|localhost:9090
Request-Header|Referer|http://localhost:9090/form.html
Request-Header|Upgrade-Insecure-Requests|1
Request-Header|User-Agent|Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:58.0) Gecko/20100101 Firefox/58.0
Parts-Count|2
Part-ContainsContents|user|fireuser
Part-ContainsContents|comment|with detailed message